src/core/ext/client_config/resolver_result.h

```diff
@@ -57,7 +57,7 @@ void grpc_resolver_result_unref(grpc_exec_ctx* exec_ctx,
                                 grpc_resolver_result* result);
 
 /// Caller does NOT take ownership of result.
```

Comment:
This is what I'm not too happy about, returning something that's not meant to be owned as a mutable.